guide/layouts/default.haml

```diff
@@ -1,12 +1,12 @@
 !!! 5
 %html{lang: "fr", "data-fr-scheme": "system"}
-  = render '/partials/head.slim'
+  != render '/partials/head.slim'
```

Comment:
from HAML docs : 

> By default, the single equals doesn’t sanitize HTML either. However, if the :escape_html option is set, = will sanitize the HTML, but != still won’t. For example, if :escape_html is set:

I could not grep `escape_html` in this repo nor in `nanoc` repo. are you sure this is necessary?

Reply:
I'm hoping it's not. For some reason it doesn't work with a plain `=` (the markup is escaped and the page displays the tags) but I'll have another look at it. See https://github.com/haml/haml/issues/1084 too.

Reply:
👌 thanks for the detail, it's fine with me to leave it as is with the bang

Reply:
I found out: https://github.com/haml/haml/commit/0322d95341cb6a59df4673fd0b5d0d02d82f9d24

Reply:
👍 thanks for digging